src/Compilers/Core/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -19,6 +19,9 @@ internal sealed class NamedPipeClientConnection : IClientConnection
         // This is a value used for logging only, do not depend on this value
         private readonly string _loggingIdentifier;
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", 
+            MessageId = "System.Runtime.InteropServices.DangerousGetHandle", 
+            Justification = "We are using DangerousGetHandle for logging only.")]
         internal NamedPipeClientConnection(NamedPipeServerStream pipeStream)
```

Comment:
That is a good question. @agocke Is there a chance we could just use an incrementing int here? The ID must be unique, but it does not need to match the actual handle, right?